components/layout/flow/root.rs

```diff
@@ -65,10 +65,7 @@ impl BoxTree {
             // From https://www.w3.org/TR/css-overflow-3/#overflow-propagation:
             // > If visible is applied to the viewport, it must be interpreted as auto.
             // > If clip is applied to the viewport, it must be interpreted as hidden.
-            viewport_scroll_sensitivity: AxesScrollSensitivity {
-                x: viewport_overflow.x.to_scrollable().into(),
-                y: viewport_overflow.y.to_scrollable().into(),
-            },
```

Comment:
I think it makes more sense to keep the `to_scrollable()` here.
But if you move it, you need to move the comment too.